src/components/text-editor/test-setup/content-generator.spec.ts

```
@@ -0,0 +1,255 @@
+                    attrs: {
+                        href: 'https://example.com',
+                        title: 'Example',
+                    },
+                },
+            ];
+
+            const state = createDocWithFormattedText(text, marks);
+
+            // Check mark attributes
+            const firstChild = state.doc.firstChild;
+            if (firstChild && firstChild.firstChild) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)